ctgan/synthesizers/ctgan.py

```diff
@@ -197,15 +196,12 @@ def _gumbel_softmax(logits, tau=1, hard=False, eps=1e-10, dim=-1):
         Returns:
             Sampled tensor of same shape as logits from the Gumbel-Softmax distribution.
         """
-        if version.parse(torch.__version__) < version.parse('1.2.0'):
```

Comment:
Once updating to torch >2.0, this problem seems. to occur again. Although not too frequently, `gumbel_softmax` sometimes returns nans.